lottie/src/main/java/com/airbnb/lottie/utils/DefaultLogger.java

```diff
@@ -0,0 +1,46 @@
+package com.airbnb.lottie.utils;
+
+import android.util.Log;
+
+import com.airbnb.lottie.L;
+
+import java.util.HashSet;
+import java.util.Set;
+
+// Default logger
+public class DefaultLogger implements LottieLogger {
+
+  /**
+   * Set to ensure that we only log each message one time max.
+   */
+  private static final Set<String> loggedMessages = new HashSet<>();
+
+
+  public void debug(String message) {
+    debug(message, null);
+  }
+
+  public void debug(String message, Throwable exception) {
+
+    if (L.DBG) {
+      Log.d(L.TAG, message, exception);
+    }
+
+  }
+
+  public void warning(String message) {
+    warning(message, null);
+  }
+
+  public void warning(String message, Throwable exception) {
+
```

Comment:
Can you remove the extra newlines you have between the function name and first line of code. Same with whitespace before closing braces?

Reply:
Sure will do